src/test/java/org/llorllale/cactoos/matchers/MatcherOfTest.java

```diff
@@ -54,26 +57,20 @@ void mismatchesFunc() {
         new Assertion<>(
             "mismatches when arg does not satisfy the predicate",
             new MatcherOf<>(
-                x -> x > 5,
-                new TextOf("Must be > 5")
+                x -> x > 7,
+                desc -> desc.appendText(
+                    new FormattedText("\"%s\"", "Must be > 7").asString()
```

Comment:
@baudoliver7 why use FormattedText here, it doesn't seem needed, neither adding extra quotes I believe